core/src/main/java/org/apache/brooklyn/feed/ssh/SshFeed.java

```diff
@@ -28,6 +28,19 @@
 import java.util.concurrent.Callable;
 import java.util.concurrent.TimeUnit;
 
+import org.slf4j.Logger;
```

Comment:
@sjcorbett send me a [interesting link]( http://stackoverflow.com/a/17194980/41195) to add Eclipse's import style to IntelliJ IDE. I hope it will be useful for you.